packages/common/container-definitions/package.json

```diff
@@ -131,12 +131,6 @@
 				"forwardCompat": false,
 				"backCompat": false
 			},
-			"InterfaceDeclaration_IContainer": {
```

Comment:
All entries should have disappeared from `broken`, right? Does the fact some remained mean this package already has changes that break those types between rc2 and rc3?

Reply:
All entries were removed, then I added back what was necessary for compilation. It does _imply_ there are type changes in RC3, but I also don't know what to make of the type tests now that we are using subpath exports.